src/hotspot/os/linux/cgroupSubsystem_linux.cpp

```diff
@@ -167,19 +178,30 @@ bool CgroupSubsystemFactory::determine_type(CgroupInfo* cg_infos,
       cg_infos[CPUACCT_IDX]._name = os::strdup(name);
       cg_infos[CPUACCT_IDX]._hierarchy_id = hierarchy_id;
       cg_infos[CPUACCT_IDX]._enabled = (enabled == 1);
+    } else if (strcmp(name, "pids") == 0) {
+      log_info(os, container)("Detected optional pids controller entry in %s", proc_cgroups);
```

Comment:
Is there a particular reason why this is level INFO? We have log_debug() or log_trace() for similar debugging in the same file.  I don't think this will need a higher level than debug.